universal-application-tool-0.0.1/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const tailwindcss = require('tailwindcss');
+
+module.exports = {
+  entry: "./app/index.js",
+  output: {
+    path: path.resolve(__dirname, 'public/javascripts/'),
+    publicPath: '/javascripts/',
+    filename: 'bundle.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: [
+          "style-loader",
+          "css-loader",
+          {
+            loader: "postcss-loader",
+            options: {
+              postcssOptions: {
+                plugins: [
+                  tailwindcss
+                ],
+              },
+            },
```

Comment:
Suggestion (non-blocking): extract the postcss loader to a `const`

Reply:
done